### PR TITLE
LPS-183308 Analyze why we are adding css using liferay-util:html-top

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
[Link to ticket](https://issues.liferay.com/browse/LPS-183308)

Remove liferay-util:html-top by importing it in main.css to follow the recommendations.

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by BarbaraCabrera** [comment]
ci:test:sf

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/_VocabulariesSelection.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
+@import 'Variables';
```
Comment (sandrodw3) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/_VocabulariesSelection.scss:1]: As the variables file is a general one and it's not related to a specific component, we can leave it lowercased

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss [hunk 2]
```diff
@@ -1,9 +1,9 @@
+@import 'Variables';
 @import 'cadmin-variables';
-@import 'variables';
 
-@import './download-spreadsheet-button';
-@import './tree';
-@import './vocabularies_selection';
+@import 'DownloadSpreadsheetButton';
+@import 'Tree';
```
Comment (sandrodw3) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss:5]: We don't need to import the Tree one here as it's already being imported in the react component 😛

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SelectFileExtension.js [hunk 3]
```diff
@@ -18,7 +18,7 @@ import React from 'react';
 import TreeFilter from './TreeFilter/TreeFilter';
 import {nodeTreeArrayMapper} from './TreeFilter/treeUtils';
 
-import '../../css/tree.scss';
+import '../../css/_Tree.scss';
```
Comment (sandrodw3) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SelectFileExtension.js:21]: As it's being imported here, I think we can avoid the `_`, and we could also rename so it matches component's name 😄

**Review by sandrodw3 — COMMENTED** [review]
Looks good overall! Just left some tiny suggestions 😄

**Review by sandrodw3 — COMMENTED** [review]
Looks good overall! Just left some tiny suggestions 😄

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SelectFileExtension.js (hunk 3 above)
Comment (sandrodw3) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SelectFileExtension.js:21]: As far as I could see, this file is being used for two components. Both components use the `TreeFilter` component, so I guess we can move the import there

**Comment by BarbaraCabrera** [comment]
ci:test:sf

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss [hunk 4]
```diff
@@ -1,9 +1,9 @@
 @import 'cadmin-variables';
 @import 'variables';
 
-@import './download-spreadsheet-button';
-@import './tree';
-@import './vocabularies_selection';
+@import 'DownloadSpreadsheetButton';
+@import 'VocabulariesSelection';
+@import 'selectFileExtension';
```
Comment (sandrodw3) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss]: This one should start with uppercase as well, as the react component 😄

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/TreeFilter/TreeFilter.js [hunk 5]
```diff
@@ -23,6 +23,8 @@ import {cancelDebounce, debounce, getOpener} from 'frontend-js-web';
 import PropTypes from 'prop-types';
 import React, {useCallback, useEffect, useMemo, useRef, useState} from 'react';
 
+import 'selectFileExtension';
```
Comment (sandrodw3) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/TreeFilter/TreeFilter.js:26]: This import is not working, here in React we need to put the full path, you need to put something like `import '../../../css/selectFileExtension.scss';` 😛

**Review by sandrodw3 — COMMENTED** [review]
Hey @BarbaraCabrera! Thanks for the changes 😄 I left some comments, the tree filter part is not working. Could you please take a look?

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/TreeFilter/TreeFilter.js (hunk 5 above)
Comment (sandrodw3) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/TreeFilter/TreeFilter.js:26]: By the way, as we moved the import to `TreeFilter` component, I think we could rename the CSS file to `TreeFilter.scss` as well, what do you think?

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**